Assets/MRTK/Examples/Common/Prefabs/HandMenu_Slider.prefab

```diff
@@ -790,7 +790,13 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   thumbRoot: {fileID: 7993839106062047049}
+  isTouchable: 0
+  snapToPosition: 0
+  thumbCollider: {fileID: 0}
```

Comment:
@RogPodge was this something missed in https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9983, or are these values okay to be unset?

Reply:
`thumbCollider ` needs to be set to Slider_Dot_geo in the prefab. The other two fields can be unset

![image](https://user-images.githubusercontent.com/39840334/168687937-a5b391e4-6038-4f5c-a959-0d0326a0ef37.png)

Reply:
Done!